src/app.ts

```diff
@@ -86,6 +67,47 @@ export default (
       .inc();
   };
 
+  const createTaskRun = (
+    name: string,
+    context: any,
+    extraParams: Array<Record<string, unknown>> = []
+  ) => {
+    var params = [
```

Comment:
Oh, one more issue (found by `npm run lint` here which suggests to use `let` over `var` any time it's possible). In this case, `params` should be `const`.

Reply:
I'll start using `npm run lint` from now on :smile:.